From deac9815a5b9b1e437b08a98915a48c0c324abb9 Mon Sep 17 00:00:00 2001
From: "Konstantin Kirik (snegovick)" <snegovick@uprojects.org>
Date: Wed, 11 Oct 2017 17:07:47 +0300
Subject: [PATCH] liblibc do not link libc statically for ARM musl

---
 src/unix/mod.rs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/unix/mod.rs b/src/unix/mod.rs
index f60985c4..d054f15b 100644
--- a/src/unix/mod.rs
+++ b/src/unix/mod.rs
@@ -237,7 +237,8 @@ cfg_if! {
     } else if #[cfg(feature = "use_std")] {
         // cargo build, don't pull in anything extra as the libstd dep
         // already pulls in all libs.
-    } else if #[cfg(any(all(target_env = "musl", not(target_arch = "mips"))))] {
+    } else if #[cfg(any(all(target_env = "musl", not(target_arch = "mips"),
+                            not(target_arch = "arm"))))] {
         #[link(name = "c", kind = "static", cfg(target_feature = "crt-static"))]
         #[link(name = "c", cfg(not(target_feature = "crt-static")))]
         extern {}
-- 
2.14.2

