--- ./src/libunwind/lib.rs	2017-10-11 21:42:12.094315904 +0300
+++ ./src/libunwind/lib.rs.new	2017-10-11 21:53:47.526509159 +0300
@@ -28,7 +28,8 @@
 #[cfg(not(target_env = "msvc"))]
 pub use libunwind::*;
 
-#[cfg(all(target_env = "musl", not(target_arch = "mips")))]
+#[cfg(all(target_env = "musl", not(target_arch = "mips"),
+    not(target_arch = "arm")))]
 #[link(name = "unwind", kind = "static", cfg(target_feature = "crt-static"))]
 #[link(name = "gcc_s", cfg(not(target_feature = "crt-static")))]
 extern {}
